ramls/request.json

```diff
@@ -135,12 +140,9 @@
             }
           }
         },
-        "copyNumbers": {
-          "type": "array",
-          "description": "Copy numbers for the item",
-          "items": {
-            "type": "string"
-          }
+        "copyNumber": {
```

Comment:
Should this be read-only?

Reply:
Following properties have been declared as read only:
* item.* (already readonly, only few properties was missed)
* requester.* (already readonly, only few properties was missed)
* proxy.* (already readonly, only few properties was missed)
* deliveryAddress.*
* pickupServicePoint.*

Reply:
Thank you